### PR TITLE
UCT/IB: Set NUMA policy for ODP memory regions.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: src/uct/ib/base/ib_md.c [hunk 1]
```diff
@@ -11,13 +11,49 @@
 
 #include <ucs/arch/atomic.h>
 #include <pthread.h>
+#include <numaif.h>
```
Comment (shamisp) [inline src/uct/ib/base/ib_md.c]: Where do we check for the availability of the headers ?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/2869/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/329/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/2273/ for details (Mellanox internal link).

File: src/uct/ib/base/ib_md.c (hunk 1 above)
Comment (yosefe) [inline src/uct/ib/base/ib_md.c]: we check the presence of the library with AC_CHECK_LIB. Do we need to check headers as well?

**Comment by yosefe** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/333/ for details.

File: src/uct/ib/base/ib_md.c (hunk 1 above)
Comment (shamisp) [inline src/uct/ib/base/ib_md.c]: For headers you have to install the *dev.rpms. So I would say you have to check both.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/2275/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/334/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/336/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/337/ for details.

File: src/uct/ib/base/ib_md.c (hunk 1 above)
Comment (yosefe) [inline src/uct/ib/base/ib_md.c]: added a check

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/2872/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/338/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/2277/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
bot:bgateNEW:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/342/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/343/ for details.

**Pull request merged**